mindjunkies/dashboard/views.py

```
@@ -1,62 +1,120 @@
+        elif status == "archived":
+            courses = Course.objects.filter(teacher=request.user, status="archived")
+            context["courses"] = courses
+            context["status"] = "Archived"
+            return render(request, "components/contents.html", context)
+        
+        elif status == "balance":
+            balance = Balance.objects.filter(user=request.user).first()
+            print(balance)
+            if not balance:
+                balance = Balance.objects.create(user=request.user, amount=0)
+            transactions = Transaction.objects.filter(user=request.user).order_by('-tran_date')  
```

Ruff (0.8.2)

Trailing whitespace

Remove trailing whitespace

(W291)